mmaction/models/localizers/bmn.py

```diff
@@ -291,8 +291,10 @@ def generate_labels(self, gt_bbox):
         for every_gt_bbox in gt_bbox:
             gt_iou_map = []
             for start, end in every_gt_bbox:
-                start = start.numpy()
-                end = end.numpy()
+                if isinstance(start, torch.Tensor):
```

Comment:
Is this a bug fix?

Reply:
Yes. Sometimes these two variables `start` and `end` are already numpy.float64 data types when parameters are passed in. It will fail in unittest but the test was passed on GitHub CI.